src/webots/Makefile

```
@@ -482,6 +482,7 @@ OTHER_SOURCES = \
   WbControllerPlugin.cpp \
   WbDataPacket.cpp \
   WbDevice.cpp \
+	WbDesktopServices.cpp \
```

Small indentation issue:
```suggestion
  WbDesktopServices.cpp \
```